### ISSUE TITLE
Missing phpdoc comment in alsoknow() method

### ISSUE THREAD

**Issue by jcvignoli** [tboothman/imdbphp#297]
Just found out that method alsoknow() in title.php is missing in its phpdoc comment the "comment" (singular) variable in @return line 1175.
This line 
`     * @return array<array{title: string, country: string, countryCode: string|null, language: string, languageCode: string|null, comments: string[]}>`
should be replaced by
`     * @return array<array{title: string, country: string, countryCode: string|null, language: string, languageCode: string|null, comment: string, comments: string[]}>`
I added "comment: string, ".
It is necessary for projects using imdbphp and phpstan.

**Comment by tboothman** [comment]
It was slightly deliberate. I didn't want it to be there, but left the comment in the actual return for backwards compatibility.
The docbloc used to say
```
* Note: This may return an empty country or comments. The original title will have a country of '' and a comment of 'original title'
* comment, year and lang are there for backwards compatibility and should not be used
* @return array array[0..n] of array[title,country,comments[]]
```
alsoknow() was the first method I fixed for 8.0 so I was trying to keep backwards compatibility. As I went on so many things had to change I started removing depricated stuff.
Explicitely talking about the field I'd wanted to remove in the comment was pretty stupid though.

It's actually impossible to have more than one 'comment'. The language used to get rolled into the comments array i think, but not any more.
![image](https://user-images.githubusercontent.com/1813336/219116026-0d94d499-735e-4b7d-a49b-655f90340b89.png)

Arguably then comment is the good field and comments is the one that should be depricated
